README.md

```diff
@@ -129,6 +129,6 @@ problematic code early.
 Author
 ------
 
-Copyright (c) 2008-2016 simplabs GmbH ([http://simplabs.com](http://simplabs.com)), released under the MIT license.
+Copyright (c) 2008-2017 simplabs GmbH ([http://simplabs.com](http://simplabs.com)), released under the MIT license.
```

Comment:
https://github.com/simplabs/excellent/graphs/contributors?from=2008-12-30&to=2009-03-01 shows first commit in 2009